diff --git a/wns/cow/wn-data-cmn.tab b/wn-data-cmn-fixed.tab
index b6d6eb2..3b13ce1 100644
--- a/wns/cow/wn-data-cmn.tab
+++ b/wn-data-cmn-fixed.tab
@@ -19839,7 +19839,7 @@
 02004874-v	cmn:lemma	回到
 02723951-v	cmn:lemma	回到
 13937918-n	cmn:lemma	回到起点
-15178842-n	cmn:lemma	回历
+15178841-n	cmn:lemma	回历
 15218551-n	cmn:lemma	回历10月
 15218798-n	cmn:lemma	回历12月
 15217443-n	cmn:lemma	回历2月
@@ -24361,7 +24361,7 @@
 03744276-n	cmn:lemma	存储器
 02671224-n	cmn:lemma	存储器存取
 03744840-n	cmn:lemma	存储设备
-15171858-n	cmn:lemma	存取时间
+15171857-n	cmn:lemma	存取时间
 13957493-n	cmn:lemma	存在
 13962360-n	cmn:lemma	存在
 02603699-v	cmn:lemma	存在
@@ -26107,7 +26107,7 @@
 03007130-n	cmn:lemma	小教堂
 13736197-n	cmn:lemma	小数
 13254011-n	cmn:lemma	小数目
-14869977-n	cmn:lemma	小斑
+14869975-n	cmn:lemma	小斑
 14286435-n	cmn:lemma	小斑
 13910019-n	cmn:lemma	小斑点
 03719560-n	cmn:lemma	小斗篷
@@ -27625,7 +27625,7 @@
 03039314-a	cmn:lemma	希伯来+的
 03039087-a	cmn:lemma	希伯来人+的
 03039314-a	cmn:lemma	希伯来人+的
-15177867-n	cmn:lemma	希伯来历
+15177866-n	cmn:lemma	希伯来历
 03039087-a	cmn:lemma	希伯来语+的
 03039314-a	cmn:lemma	希伯来语+的
 02930765-a	cmn:lemma	希伯莱语+的
@@ -36107,7 +36107,7 @@
 02899486-a	cmn:lemma	教会+的
 03029197-n	cmn:lemma	教会塔
 03028785-n	cmn:lemma	教会帽子
-15173065-n	cmn:lemma	教会年
+15173064-n	cmn:lemma	教会年
 15181718-n	cmn:lemma	教会年历
 03263338-n	cmn:lemma	教会服装
 13247818-n	cmn:lemma	教会的收入
@@ -44567,7 +44567,7 @@
 03669367-n	cmn:lemma	污水道
 14956661-n	cmn:lemma	污泥
 00422620-a	cmn:lemma	污浊的
-14869976-n	cmn:lemma	污点
+14869975-n	cmn:lemma	污点
 04694441-n	cmn:lemma	污点
 14856752-n	cmn:lemma	污物
 03212114-n	cmn:lemma	污物碾碎器
@@ -55995,7 +55995,7 @@
 03522003-n	cmn:lemma	竞技场
 08497107-n	cmn:lemma	竞技场
 13943740-n	cmn:lemma	竞技场
-15171739-n	cmn:lemma	竞技状态不佳的日子
+15171738-n	cmn:lemma	竞技状态不佳的日子
 01168569-n	cmn:lemma	竞赛
 07456188-n	cmn:lemma	竞赛
 07458453-n	cmn:lemma	竞赛
@@ -57898,7 +57898,7 @@
 03743902-n	cmn:lemma	纪念堂
 03745571-n	cmn:lemma	纪念巨碑
 02864165-a	cmn:lemma	纪念性+的
-15171147-n	cmn:lemma	纪念日
+15171145-n	cmn:lemma	纪念日
 15200661-n	cmn:lemma	纪念日
 03074380-n	cmn:lemma	纪念柱
 03743902-n	cmn:lemma	纪念碑
@@ -62629,7 +62629,7 @@
 03757604-n	cmn:lemma	节拍器
 03141756-a	cmn:lemma	节拍器+的
 00982275-a	cmn:lemma	节拍或曲子微急+的
-15171146-n	cmn:lemma	节日
+15171145-n	cmn:lemma	节日
 00517728-n	cmn:lemma	节日
 15183428-n	cmn:lemma	节日
 12701178-n	cmn:lemma	节柄科
@@ -66350,7 +66350,7 @@
 02296824-a	cmn:lemma	规定+的
 15233778-n	cmn:lemma	规定时间
 13580909-n	cmn:lemma	规定浓度
-15168570-n	cmn:lemma	规定的睡觉时间
+15168569-n	cmn:lemma	规定的睡觉时间
 02004838-a	cmn:lemma	规定费用+的
 13580909-n	cmn:lemma	规度
 03384706-n	cmn:lemma	规整式园林
@@ -71171,7 +71171,7 @@
 02717538-a	cmn:lemma	选举+的
 02454835-v	cmn:lemma	选举中获得
 10760340-n	cmn:lemma	选举人
-15172882-n	cmn:lemma	选举日
+15172881-n	cmn:lemma	选举日
 05186306-n	cmn:lemma	选举权
 00800121-n	cmn:lemma	选举活动
 02400637-v	cmn:lemma	选出
@@ -75831,7 +75831,7 @@
 02262984-a	cmn:lemma	雾化+的
 02754103-n	cmn:lemma	雾化器
 03375171-n	cmn:lemma	雾号
-15176162-n	cmn:lemma	雾月
+15176161-n	cmn:lemma	雾月
 11458314-n	cmn:lemma	雾气
 02549569-a	cmn:lemma	雾气覆盖+的
 00403052-r	cmn:lemma	雾浓+地
